.changeset/fluffy-zebras-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/router': patch
+---
+
+Add an optional `fallbackRoute` param to the Routes and Router class.
```

Comment:
Update to describe the new name and options bag.

Reply:
Done